samples/vsphere/appliances/local_accounts.py

```
@@ -0,0 +1,122 @@
+                                            session=session)
+        self.local_accounts = self.client.appliance.LocalAccounts
+
+    def run(self):
+        """
+        Running the workflow for Local accounts.
+        It creates local account and updates its default security settings.
+        Deletes the created local account at the end.
+        """
+
+        print("Listing available accounts")
+        self.list_accounts()
+
+        print("Create local account for yourself")
+        local_user = self.create_local_account()
+
+        print("Updating the local accounts security settings")
+        self.update_local_account_security(local_user)
+
+        print("Get information for specific account")
+        self.get_account_info()
+
+        print("Deleting the local account")
+        self.delete_local_account(local_user)
+
+    def get_account_info(self):
+        try:
+            username = input("username ::")
+            pprint(self.local_accounts.get(username))
+        except NotFound as e:
+            print("Local Account mentioned is not found")
+
+    def list_accounts(self):
+        pprint(self.local_accounts.list())
+
+    def create_local_account(self):
+        account_created = False
+        try:
+            config = self.local_accounts.Config()
+            print("The following are minimum details required to create local account")
+            username = input("username of local account ::")
+            config.password = getpass("password ::")
+            print("Roles can be operator, admin, superAdmin.")
+            config.roles = [input("role :: ")]
+            config.full_name = input("Full name of user ::")
+
+            self.local_accounts.create(username=username, config=config)
+            print("Listing available accounts after creation of " + username)
+            self.list_accounts()
+            account_created = True
+        except AlreadyExists as e:
+            print("local account is already present")
```

Re-raise exceptions instead of print, so the issues are more obvious.